codecarbon/input.py

```diff
@@ -64,6 +70,23 @@ def get_country_emissions_data(self, country_iso_code: str) -> Dict:
         :param country_iso_code: ISO code similar to one used in file names
         :return: emissions in lbs/MWh and region code
         """
-        with open(self.country_emissions_data_path(country_iso_code)) as f:
-            country_emissions_data: Dict = json.load(f)
-        return country_emissions_data
+        try:
+            with open(self.country_emissions_data_path(country_iso_code)) as f:
+                country_emissions_data: Dict = json.load(f)
+            return country_emissions_data
+        except KeyError:
```

Comment:
In which case could we get a `KeyError`?

Reply:
I just added a comment specifying. The KeyError comes from line 39, when the filepath of the country emissions data file is looked up from the config object. If there is no file, it raises a KeyError